### PR TITLE
Fix pipeline tests

### PR DESCRIPTION
**Pull request by denysop**

### CODE REVIEW

File: .github/workflows/react-native-code-push-ci.yml [hunk 1]
```diff
@@ -27,12 +27,23 @@ jobs:
...
+          java-version: '11'
       - name: Package Installation
-        run: npm install
+        run: npm i
+      - name: Install react-native
+        run: npm install -g react-native
+      - name: Setup Android tests
+        run: npm run build:tests && npm run test:setup:android
+      - name: Remove react-native
+        run: npm remove -g react-native
```
Comment (yevheniy-solovey-ms) [inline .github/workflows/react-native-code-push-ci.yml:42]: Why do we need to remove the react-native?

File: .github/workflows/react-native-code-push-ci.yml [hunk 2]
```diff
@@ -44,6 +55,8 @@ jobs:
         run: npm install
       - name: Install react-native-cli
         run: npm install react-native-cli
+      - name: Install reacti-native
+        run: npm install react-native
```
Comment (yevheniy-solovey-ms) [inline .github/workflows/react-native-code-push-ci.yml:59]: And then install it here again

Comment (denysop) [reply .github/workflows/react-native-code-push-ci.yml:59]: It is iOS pipeline. There is a separate task to make if functional so you can ignore this part for now.

File: .github/workflows/react-native-code-push-ci.yml (hunk 1 above)
Comment (denysop) [reply .github/workflows/react-native-code-push-ci.yml:42]: It is a workaround for the next error: ```ReferenceError: SHA-1 for file /metro-runtime/src/polyfills/require.js) is not computed.```

Taken from here: https://github.com/facebook/metro/issues/330#issuecomment-572917876

**Review by yevheniy-solovey-ms — APPROVED** [review]

**Pull request merged**